vms/proposervm/batched_vm_test.go

```diff
@@ -30,6 +30,7 @@ func TestCoreVMNotRemote(t *testing.T) {
 	// if coreVM is not remote VM, a specific error is returned
 	require := require.New(t)
 	_, _, proVM, _, _ := initTestProposerVM(t, time.Time{}, 0) // enable ProBlks
+	defer require.NoError(proVM.Shutdown(context.Background()))
```

Comment:
This shuts down the VM immediately - as the arguments for `require.NoError` are evaluated at the time of the `defer` statement - and then `require.NoError` is executed at the end of the function.,

Reply:
Reverted to wrap the `defer require` into a `defer func(){require...}` to avoid early arguments evaluation.